sysutils/afscompress/Portfile

```diff
@@ -45,6 +55,11 @@ pre-fetch {
     }
```

Comment:
@ylluminarious are you sure that it won't work before 10.6?

I see that it has some code to make it works: https://github.com/RJVB/afsctool/blob/master/src/afsctool.c#L73-L77

Reply:
> @ylluminarious are you sure that it won't work before 10.6?
> 
> I see that it has some code to make it works: https://github.com/RJVB/afsctool/blob/master/src/afsctool.c#L73-L77

I'll test on my old 10.5 machines - both Intel and PPC - when I can find some time.

For now though, let's go ahead and merge, as this is a nice addition. Thanks as always Kirill!